src/Symfony/Component/Runtime/README.md

```diff
@@ -0,0 +1,104 @@
+Runtime Component
+=================
+
+Symfony Runtime enables decoupling apps from global state.
+
+Getting Started
+---------------
+
+```
+$ composer require symfony/runtime
+```
+
+RuntimeInterface
+----------------
+
+The core of this component is the `RuntimeInterface` which describes a high-order
+runtime logic.
+
+It is designed to be totally generic and able to run any application outside of
+the global state in 6 steps:
+
+ 1. your front-controller returns a closure that wraps your app;
+ 2. the arguments of this closure are resolved by `RuntimeInterface::resolve()`
+    which returns a `ResolvedAppInterface`. This is an invokable with zero
+    arguments that returns whatever object of yours represents your app
+    (e.g a Symfony kernel or response, a console application or command);
```

Comment:
An Sf response might be correct, but it seems confusing in this context, as I would not expect it to represent an "app".

In fact I have a bit of a hard time understanding this whole explanation, because to me "the app" is primarily the bits of code which are given in the examples below, not the Sf kernel or Command...